src/components/GeneratorHeader/GeneratorHeader.tsx

```diff
@@ -1,13 +1,18 @@
 import Link from 'next/link';
 import React from 'react';
 import { linkToGenerator, linkToSearch } from '../../helpers/links';
+import styles from './GeneratorHeader.module.scss';
 
 function GeneratorHeader() {
     return (
         <header>
-            <nav>
-                <Link href={linkToGenerator()}>Generator</Link>
-                <Link href={linkToSearch()}>Search</Link>
+            <nav className={styles.root}>
+                <Link className={[styles.activeTab, styles.tab].join(' ')} href={linkToGenerator()}>
```

Comment:
Please use `classnames` library to apply multiple classes.
```suggestion
                <Link className={cn(styles.activeTab, styles.tab)} href={linkToGenerator()}>
```

and you will need import at the top:
```js
import cn from 'classnames';
```